docs/section5/Rule5-33.md

```diff
@@ -0,0 +1,43 @@
+
+# Envelope - Rule 5-33  
+
+**Rule ID:** 5-33  
+**Rule Description:** Automatically controlled fenestration shading devices must be modeled in the proposed design the same as in user model.  
+**Rule Assertion:** P-RMR subsurface:has_automatic_shades = U-RMR subsurface:has_automatic_shades  
+**Appendix G Section:** Section G3.1-5(a)(4) Building Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:**  None
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** None  
+
+**Manual Checks:** None  
+**Evaluation Context:**  Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. match_data_element()
+  2. get_opaque_surface_type()
+
```

Comment:
I don't think we need this function

Reply:
Wall check and the function is now removed.